Codeinwp/templates-patterns-collection
# Fix design issues

### Summary
In this PR I address the following issues:
- Remove the top header for certain steps
- Make the next button sticky
- Added missing UTM tags
- Change the look of the logo component
- Added missing box shadow for the settings sidebar
- Keep the same aspect ratio for starter site cards

Note: After discussing this with Hardeep, the loading conditions for Onboarding were changed. The onboarding doesn't check for Neve's new users anymore. In order to load, here's what it checks for:
- User can install plugins
- Starter sites are not disabled in white-label module and
- The current theme is Neve or a child theme of it and
- It's the first installation of TPC and
- The onboarding process was not finished

### Test instructions
- Install and activate Neve
- Install this version of TPC and activate it
- The onboarding should start as soon as you activate TPC
- Check and see if you can find any issues with the implementation of the previously mentioned changes



Closes #288.
